src/pages/ReimbursementAccount/BeneficialOwnersStep.js

```diff
@@ -80,7 +83,15 @@ class BeneficialOwnersStep extends React.Component {
     }
 
     removeBeneficialOwner(beneficialOwner) {
-        this.setState(prevState => ({beneficialOwners: _.without(prevState.beneficialOwners, beneficialOwner)}));
+        this.setState((prevState) => {
+            const beneficialOwners = _.without(prevState.beneficialOwners, beneficialOwner);
+
+            // We set 'beneficialOwners' to null first because we don't have a way yet to replace a specific property without merging it.
+            // We don't use the debounced function because we want to make both function calls.
+            updateReimbursementAccountDraft({beneficialOwners: null});
```

Comment:
Hmm... yeah I think we should solve this before adding too many cases. Otherwise at the minimum we should link directly to this code in the issue opened [here](https://github.com/Expensify/Expensify/issues/176419). Honestly, I thought it would take some more time before someone would copy this bad example 😓

Reply:
Agreed! If this PR gets merged before, I'll link the code to the issue you mentioned so we know to update it later!

Reply:
Thanks!